src/table/wcsUtils.cc

```diff
@@ -91,35 +92,35 @@ void updateRefCentroids(geom::SkyWcs const &wcs, ReferenceCollection &refList) {
     }
 }
 
-Eigen::Matrix2f calculateCoordCovariance(geom::SkyWcs const& wcs, lsst::geom::Point2D center, Eigen::Matrix2f err) {
+Eigen::Matrix2f calculateCoordCovariance(geom::SkyWcs const &wcs, lsst::geom::Point2D center,
+                                         Eigen::Matrix2f err) {
+    if (!std::isfinite(center.getX()) || !std::isfinite(center.getY())) {
```

Comment:
I think this would be the end result anyway, but I agree it's better to just skip going forward in this situation.

Reply:
No, if the center is non-finite, the computed `skyCenter` will be NaN and the call to `geom::makeSkyWcs` will raise because it's not a valid WCS.